serve-web/src/Entity/Order.php

```diff
@@ -67,25 +63,25 @@ abstract public function isOrderValid(): bool;
     private ?string $appointmentType = null;
 
     /**
-     * Date order was created in DC database
+     * Date order was created in DC database.
      */
     #[ORM\Column(name: 'created_at', type: 'datetime')]
-    private DateTime $createdAt;
+    private \DateTime $createdAt;
```

Comment:
We probably need to agree what convention we want to use for root imports, as I've been doing the opposite and importing everything